File: Two Pointers/2348. Number of Zero-Filled Subarrays.py
class Solution:
    def zeroFilledSubarray(self, nums: List[int]) -> int:
        res = 0
        consective = 0
        for i in nums:
            if i != 0: 
                consective = 0
            else:
                consective += 1 
                res += consective
        return res

File: Two Pointers/2348. Number of Zero-Filled Subarrays.cpp
class Solution {
public:
    long long zeroFilledSubarray(vector<int>& nums) {
        long long res = 0;
        for (int i = 0, j = 0; i < nums.size(); ++i) {
            if (nums[i] != 0)
                j = i + 1;
             res += i - j + 1;
        }
        return res;
    }
};